docs/user_manual/components.md

```diff
@@ -147,6 +147,87 @@ $$
    \end{eqnarray}
 $$
 
+### Asym Line
```

Comment:
because this component is probably fairly rare compared to regular `line`s, `link`s and `transformer`s (and maybe even `generic_branch`es), can you maybe put them below those? that way, the flow of the branch components remains the same as it was before, which will help users in looking things up in the documentation